cmd/nerdctl/ipfs_registry_up.go

```diff
@@ -36,8 +37,17 @@ const ipfsRegistryContainerName = "ipfs-registry"
 
 func newIPFSRegistryUpCommand() *cobra.Command {
 	var ipfsRegistryUpCommand = &cobra.Command{
-		Use:           "up",
-		Short:         "start registry as a background container \"ipfs-registry\", backed by the current user's IPFS API",
+		Use:   "up",
+		Short: "start or restart registry as a systemd service, backed by the current user's IPFS API",
+		Long: `start or restart registry as a systemd service, backed by the current user's IPFS API
+
+This command requires "nerdctl ipfs registry serve" command is registered as a systemd service named "nerdctl-ipfs-registry-serve".
+In rootless mode, use "containerd-rootless-setuptool.sh install-nerdctl-ipfs-registry-serve" to install it.
+
+Instead of using this command, you can also directly launch the registry using "nerdctl ipfs registry serve".
+
+Use "nerdctl ipfs registry down" to stop the registry service launced by "nerdctl ipfs registry up".
+`,
```

Comment:
Maybe we can just drop the up/down command and let the user install the systemd unit by themselves